src/pages/HomePage.js

```diff
@@ -20,24 +20,29 @@ const areasTablet = `
     header header
     info image
 `
+const StyleHeader = styled.header`
+    background-color: red;
+`
 
 const HomePage = () => (
     <Composition
         areas={areasMobile}
         areasMd={areasTablet}
         gutter={2}
         padding={2}
+        templateColsMd="40% auto"
+        gap={2}
```

Comment:
`gap` is the same as `gutter`. Please remove the `gap` prop.

Could you please point me to where have you found the `gap` prop being used? It's somewhere from the documentation, right?

Reply:
'gap' is right =) 'gutter' is wront :P